ansible/inventory.yml

```diff
@@ -146,6 +146,9 @@ hosts:
         aix71-ppc64_be-4:  
             ip: 169.48.22.51
             server_jobs: 6
+        aix73-ppc64_be-1:
+            ip: 163.68.64.114
+            server_jobs: 6
```

Comment:
I don't suppose we know if this is a sensible value for the new machine? I guess we can fix up later if not.

Reply:
+1

Reply:
The machine should be similar to our other machines as we asked for the same specs.